dts/riscv/ite/it82xx2.dtsi

```diff
@@ -525,6 +525,10 @@
 					 NO_FUNC  0xf03e10 NO_FUNC  0xf02032>;
 			func3-en-mask = <0x01     0x20     0x20     0
 					 0        0x08     0        0x01    >;
+			func3-ext =     <NO_FUNC 0xf02032  0xf02032 NO_FUNC
```

Comment:
As a follow-up can you move GPE0 and GPE7 setting from func3-gcr to func3-ext?

Reply:
OK, I will create another PR.